packages/twenty-front/src/modules/command-menu/components/CommandMenu.tsx

```diff
@@ -634,6 +634,7 @@ export const CommandMenu = () => {
                   selectableListId="command-menu-list"
                   selectableItemIdArray={selectableItemIds}
                   hotkeyScope={AppHotkeyScope.CommandMenu}
+                  selectFirstItem={true}
```

Comment:
why don't you set the selectedItem in the state instead? When this is not already set ofc. It would avoid adding such behavior

Reply:
It was in case we need this kind of behavior elsewhere also. But I can set the state directly inside the command menu. I don't have a strong opinion on this.